common/tari_features/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "tari_features"
+description = "Compilable features for Tari applications"
+authors = ["The Tari Development Community"]
+repository = "https://github.com/tari-project/tari"
+homepage = "https://tari.com"
+readme = "README.md"
+license = "BSD-3-Clause"
+version = "0.0.1"
```

Comment:
This is probably up for discussion, but I think we should version match here with the rest of the Tari code base. 
at least for now.

Reply:
That makes sense to me.